tests/pom.xml

```diff
@@ -36,6 +36,29 @@
           <providerHint>junit5</providerHint>
         </configuration>
       </plugin>
+       <plugin>
+		   <groupId>org.eclipse.tycho</groupId>
+		   <artifactId>target-platform-configuration</artifactId>
+		   <version>${tycho.version}</version>
+		   <configuration>
+		      <!-- dependency versions should be consistent with the versions in target-platform.target -->
```

Comment:
These two are already in target-platform.target.  Do we know why we also need them here as "extraRequirements"?   Is there an example at least we're working from?

Reply:
Per our conversation. This is needed to run from the command line as the Eclipse runtime instance that is created has only very basic functionality. A comment was added to mention this.